### PR TITLE
Add a switch for the Apple smart banner

### PR DESCRIPTION
**Pull request by marialivia16**
## What does this change?

Adds a switch for using the Apple's [smart banner](https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html). If the switch is on and it can be showed (iOS device and Safari) then we do not show the custom made banner we have. This work is part of this [idea](https://docs.google.com/document/d/1-01OozcbnIVQY8yh37jTnbNbtkGjTjy5zULEQskxKqs/edit#).
## Request for comment

@stephanfowler

### CODE REVIEW

File: common/app/views/fragments/metaData.scala.html [hunk 1]
```diff
@@ -98,6 +98,10 @@
 <meta name="theme-color" content="#005689">
 <meta name="msapplication-TileImage" content="@Static("images/favicons/windows_tile_144_b.png")" />
 
+@if(SmartAppBanner.isSwitchedOn) {
+    <meta name="apple-itunes-app" content="app-id=409128287, app-argument=@page.metadata.webUrl">
```
Comment (TBonnin) [inline common/app/views/fragments/metaData.scala.html]: I think there is a configuration value for the app id.

**Comment by TBonnin** [comment]
The branch is probably not up-to-date with master.
Other than that and a tiny comment, lgtm 👍

**Comment by stephanfowler** [comment]
👍 LGTM

File: common/app/views/fragments/metaData.scala.html (hunk 1 above)
Comment (marialivia16) [inline common/app/views/fragments/metaData.scala.html]: Good call, I changed it and I'm going to merge this.

**Pull request merged**